### PR TITLE
Fixes #176 Improve tile of Open Source Licenses

### LINKED ISSUE #176: Improve title of Open Source Licenses
#### Software and hardware versions 
Moto G5 Plus, Android 9.

#### Problem description
Title of open source licenses screen is not correct on android 9, 

#### Steps to reproduce the problem
1. In the overflow menu from the main screen, tap 'about'
2. Tap 'open source libraries / licenses'

See that the menu shows the file URI.

#### Expected behavior
Should be `Open Source Licenses` instead

#### Other information 
Screenshot:
image

This bus is similar to https://github.com/opendatakit/collect/issues/2397.

### PR DESCRIPTION
**Pull request by rohap97**
Closes #176 

#### What has been done to verify that this works as intended?
Build run on the phone

#### Why is this the best possible solution? Were any other approaches considered?
This is the only solution possible and even the shortest. The approach was just to add a String Variable and then using it to setTitle on getSupportActionBar. 

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew pmd checkstyle lint findbugs` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

File: share_app/src/main/java/org/odk/share/tasks/UploadJob.java [hunk 1]
```diff
@@ -340,6 +340,9 @@ private void sendBlankForm(Cursor cursor) {
                 sbResult.append(getContext().getString(R.string.id, formId) + " " +
                         getContext().getString(R.string.success, getContext().getString(R.string.blank_form_count,
                                 getContext().getString(R.string.sent))));
+            } else {
```
Comment (lakshyagupta21) [inline share_app/src/main/java/org/odk/share/tasks/UploadJob.java]: Why added this change?

**Comment by iadeelzafar** [comment]
@rohap97 Whenever making a PR that has some UI changes, try to add a Screenshot/GIF showing the changes. It makes it easy to review the PR.

File: share_app/src/main/java/org/odk/share/tasks/UploadJob.java (hunk 1 above)
Comment (rohap97) [reply share_app/src/main/java/org/odk/share/tasks/UploadJob.java]: @lakshyagupta21 Sir, This resolves the empty dialog box error, issue number #157

**Comment by rohap97** [comment]
@iadeelzafar I will that in mind. Thanks for the help!

**Comment by shobhitagarwal1612** [comment]
@rohap97 You have already claimed to work on 2 issues. Please do not claim any more until those two are resolved or you have unclaimed to work on them so that others can do so. This gives everyone an equal opportunity. Moreover, you weren't assigned by the @opendatakit-bot :robot: so I hope you understand this. 

Closing this PR

**Comment by rohap97** [comment]
@shobhitagarwal1612 Sir, I am sorry. I did not know about the 2 issue policy. I will make sure this doesn't happen again.

**Pull request closed without merging**